### DISCUSSION TITLE
Property 't' does not exist on type

### DISCUSSION THREAD

**Discussion by fabio-C** [fnando/i18n-js#672] [Q&A]
I got this Typescript error when I use this library in react-native.

Do you know a solution?

Details and core are here: https://stackoverflow.com/questions/73946511/i18n-js-property-t-does-not-exist-on-type-typeof-import

**Comment by fabio-C** [comment]
Solved downgrading to i18n-js 3.8

**Reply by fnando** [reply]
If your post was based on v4, then looks like you're using the wrong api. You need to instantiate I18n before you can use it.

```ts
import { I18n } from "i18n-js";
import en from "./locales/en.json";

const i18n = new I18n();
// additional setup here

i18n.store(en);
```
